### PR TITLE
Rename test file

### PR DESCRIPTION
**Pull request by oskarscholander**
And rename test file.

This updates rule-tester to 8.13.0

### CODE REVIEW

**Comment by Kalle-Rei** [comment]
Thanks for also running the tests and making sure they catch stuff 😉 
Screenshot 2024-11-07 at 13 44 02

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "@types/node": "^22.7.9",
     "@typescript-eslint/eslint-plugin": "^7.8.0",
     "@typescript-eslint/parser": "^7.8.0",
-    "@typescript-eslint/rule-tester": "7.5.0",
+    "@typescript-eslint/rule-tester": "^8.13.0",
```
Comment (dennis-meitner) [inline package.json:25]:
```suggestion
    "@typescript-eslint/rule-tester": "8.13.0",
```

Comment (dennis-meitner) [reply package.json:25]: And you want to re-run yarn and get an updated lock file after this

**Review by dennis-meitner — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (oskarscholander) [reply package.json]: Removed the update of rule-tester. It sucked.

**Pull request merged**